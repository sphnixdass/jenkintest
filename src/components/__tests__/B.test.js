import React from 'react';
import { shallow, mount } from 'enzyme';

describe('<B />', () => {
  let B;
  let makeShallowSubject;
  let makeSubject;

  beforeEach(() => {
    B = require('../B').default;

    makeShallowSubject = () => shallow(<B />);
    makeSubject = () => mount(<B />);
  });

  it('should match shallow snapshot', () => {
    const subject = makeShallowSubject();

    expect(subject.html()).toMatchSnapshot();
  });

  it('should match snapshot', () => {
    const subject = makeSubject();

    expect(subject.html()).toMatchSnapshot();
  });
});