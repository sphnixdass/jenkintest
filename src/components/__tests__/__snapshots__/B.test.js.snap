// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<B /> should match shallow snapshot 1`] = `"<nav><a href=\\"#\\">I am link 0.</a><a href=\\"#\\">I am link 1.</a><a href=\\"#\\">I am link 2.</a></nav>"`;

exports[`<B /> should match snapshot 1`] = `"<nav><a href=\\"#\\"><!-- react-text: 3 -->I am link <!-- /react-text --><!-- react-text: 4 -->0<!-- /react-text --><!-- react-text: 5 -->.<!-- /react-text --></a><a href=\\"#\\"><!-- react-text: 7 -->I am link <!-- /react-text --><!-- react-text: 8 -->1<!-- /react-text --><!-- react-text: 9 -->.<!-- /react-text --></a><a href=\\"#\\"><!-- react-text: 11 -->I am link <!-- /react-text --><!-- react-text: 12 -->2<!-- /react-text --><!-- react-text: 13 -->.<!-- /react-text --></a></nav>"`;
